core/src/test/scala/filodb.core/memstore/TimeSeriesMemStoreSpec.scala

```diff
@@ -180,14 +180,12 @@ class TimeSeriesMemStoreSpec extends FunSpec with Matchers with BeforeAndAfter w
 
   it("should handle errors from ingestStream") {
     memStore.setup(dataset1, 0, TestData.storeConf)
-    var err: Throwable = null
-
     val errStream = Observable.fromIterable(groupedRecords(dataset1, linearMultiSeries()))
                               .endWithError(new NumberFormatException)
-    val fut = memStore.ingestStream(dataset1.ref, 0, errStream, s) { ex => err = ex }
-    fut.futureValue
-
-    err shouldBe a[NumberFormatException]
+    val fut = memStore.ingestStream(dataset1.ref, 0, errStream, s)
+    ScalaFutures.whenReady(fut.failed) { e =>
```

Comment:
Usually the 'ScalaFutures' part is not needed as it is already mixed in to the trait.